### PR TITLE
MCS-1814 [Webenabled] Add task descriptions 

### PR DESCRIPTION
**Pull request by rartiss55**
Should appear in the UI right under "Scene Name"

Descriptions are matched against the scene name in the webenabled/scenes directory -- left a note about the naming convention/enum matching in mcs_task_desc.py (if naming convention isn't followed, "N/A" is displayed for task description). 

Note that this is branched from the MCS-1945 branch.

### CODE REVIEW

File: webenabled/scenes/interactive_asymmetric_tool_use_0001_10.json [hunk 1]
```diff
@@ -510,7 +510,7 @@
       "z": 12
     }
   ],
-  "name": "interactive_tool_use_0001_10",
+  "name": "interactive_asymmetric_tool_use_0001_10",
```
Comment (rartiss55) [inline webenabled/scenes/interactive_asymmetric_tool_use_0001_10.json:513]: All the tool_use scenes had to be renamed to display the accurate task description/differentiate between symmetric, asymmetric, and tool choice.

File: webenabled/templates/mcs_page.html [hunk 2]
```diff
@@ -44,6 +44,8 @@
 
         div#scene-nav-info {
             flex: 1;
+            display: flex;
```
Comment (rartiss55) [inline webenabled/templates/mcs_page.html:47]: The CSS stuff here was to address a weird bug with the left panel resizing upon a newly selected scene (should be more consistent now).

File: webenabled/templates/mcs_page.html [hunk 3]
```diff
@@ -466,16 +481,15 @@ <h1>Machine Common Sense</h1>
...
+            document.getElementById("goal-desc").style.display = "inline"
 
-                let goalDesc = document.getElementById('goal-desc');
-                goalDesc.innerHTML = data.goal.description;
-            }
+            let goalDesc = document.getElementById('goal-desc');
+            goalDesc.innerHTML = data.goal.description;
         } else {
             console.warn("description not in goalInfo");
+            if(hasNoDesc) {
```
Comment (rartiss55) [inline webenabled/templates/mcs_page.html:490]: Addressing another minor bug here with the goal info display.

**Review by ThomasSchellenbergNextCentury — APPROVED** [review]

**Review by phile-caci — APPROVED** [review]
LGTM!

**Pull request merged**